src/components/routes/qdm/TestCaseRoutes.tsx

```diff
@@ -60,10 +60,17 @@ const TestCaseRoutes = () => {
           })
           .catch((err) => {
             setContextFailure(true);
-            setErrors((prevState) => [
-              ...prevState,
-              "An error occurred, please try again. If the error persists, please contact the help desk",
-            ]);
+            setErrors((prevState) =>
+              prevState.length === 1 &&
```

Comment:
Looks like when measure.groups are empty then `convertToCqmMeasure` will throw an exception, if that is the case, Do you think we could skip the `convertToCqmMeasure` altogether when Population Criteria is not set ?

convertToCqmMeasure requires relevant DataElements, and Relevant DataElements requires Population Criteria

Reply:
Also we are only removing duplicate error exception only when prevState[0] has that particular error about PC, but in the following screenshot, its the 2nd error message and not first. 

![image](https://github.com/MeasureAuthoringTool/madie-patient/assets/57495404/3f555c0c-0650-4713-9008-63164658de9e)